src/main/java/com/sprint/mission/discodeit/service/jcf/JCFChannelService.java

```diff
@@ -0,0 +1,199 @@
+            System.out.printf("'%s' 채널의 이름이 '%s' 으로 변경되었습니다.%n",
+                    channel.getChannelName(), newName);
+            channel.updateChannelName(newName);
+        } else {
+            System.out.printf("'%s' 은 '%s' 채널 주인이 아닙니다.%n",
+                    user.getUserName(), channel.getChannelName());
+        }
+    }
+
+    @Override
+    public void deleteChannel(User user, Channel channel) {
+        if(channel == null) {
+            System.out.println("채널이 null 입니다.");
+            return;
+        }
+        if(user.getStatus().equals(User.UserStatus.DEACTIVE)) {
+            System.out.printf("'%s'는 이미 비활성 상태입니다. 따라서 '%s' 채널을 삭제할 수 없습니다.%n", user.getUserName(), channel.getChannelName());
+            return; // 메서드 종료
+        }
+        if(!channel.getUsers().contains(user)) {
+            System.out.printf("'%s'는 '%s' 채널에 존재하지 않아서 권한이 없습니다.%n", user.getUserName(), channel.getChannelName());
+            return;
+        }
+
+        if (!user.equals(channel.getHostUser())) {
+            System.out.printf("'%s'는 '%s' 채널의 주인이 아니어서 지울 수 있는 권한이 없습니다.%n", user.getUserName(), channel.getChannelName());
+            return;
+        }
+
+        System.out.printf("채널 삭제: %s%n", channel.getChannelName());
+        channel.getMessages()
+                .stream()
+                .map(message -> {
+                    message.getUser().removeMessage(message);
+                    return message;
+                });
+
+        channel.getUsers()
+                .stream()
+                .map(u -> {
+                    u.removeChannel(channel);
+                    return u;
+                });
+
+        channel.clearUsers();
+        channel.clearMessages();
+        data.remove(channel);
+    }
+
+    @Override
+    public void updateHostUser(User oldHostUser, Channel channel, User newHostUser) {
+        if(channel == null) {
+            System.out.println("채널이 null 입니다.");
+            return;
+        }
+        if(oldHostUser.getStatus().equals(User.UserStatus.DEACTIVE)) {
+            System.out.printf("'%s'는 비활성 상태이므로 권한이 없습니다.%n", oldHostUser.getUserName());
+            return;
+        }
+
+        if(newHostUser.getStatus().equals(User.UserStatus.DEACTIVE)) {
+            System.out.printf("'%s'는 비활성 상태이므로 권한이 없습니다.%n", newHostUser.getUserName());
+            return;
+        }
+
+        if(!channel.getUsers().contains(newHostUser)) {
+            System.out.printf("'%s'는 '%s' 채널에 존재하지 않아서 권한이 없습니다.%n", newHostUser.getUserName(), channel.getChannelName());
+            return;
+        }
+        if (channel.getHostUser().equals(oldHostUser)) {
+            System.out.printf("'%s' 채널 주인을 변경합니다. 새 주인: '%s'%n",
+                    channel.getChannelName(), newHostUser.getUserName());
+            channel.updateHostUser(newHostUser);
+        } else {
+            System.out.printf("'%s' 은 '%s' 채널 주인이 아닙니다.%n",
+                    oldHostUser.getUserName(), channel.getChannelName());
+        }
```

Comment:
```suggestion
        if (!channel.getHostUser().equals(oldHostUser)) {
                  System.out.printf("'%s' 은 '%s' 채널 주인이 아닙니다.%n",
              oldHostUser.getUserName(), channel.getChannelName());
        } 
        System.out.printf("'%s' 채널 주인을 변경합니다. 새 주인: '%s'%n",
                    channel.getChannelName(), newHostUser.getUserName());
            channel.updateHostUser(newHostUser);
```
이런식으로 순서를 바꿔보세요. 훨씬 가독성 좋을겁니다.